validator/js/chromeextension/background.js

```diff
@@ -95,6 +98,20 @@ function getNumberOfWarnings(errors) {
   return getErrorSeverityCounts(errors)['WARNING'];
 }
 
+/**
+ * Inspects ValidationResults to discover if in Dev Mode.
+ *
+ * @param {!Object<!ValidationResult>} validationResult
+ * @return {boolean}
+ */
+function isInDevMode(validationResult) {
+  if ((validationResult.errors.length == 1) &&
+      (validationResult.errors[0].code == 'DEV_MODE_ONLY')) {
+    return true;
+  }
+  return false;
```

Comment:
```
if (condition) {
  return true;
}
return false;
```

is an antipattern. Just use `return condition;`

Reply:
Done.